src/app/app.component.js

```diff
@@ -0,0 +1,30 @@
+import { Component, ViewEncapsulation } from '@angular/core';
+
+import style from './app.style.scss';
+import template from './app.template.html';
+
+
+export const NAVIGATION_ROUTES = [
```

Comment:
I suppose that those routes are hardcoded (they neither aren`t  ng routes  nor related with ng routing) and they mainly used to build navigation. Moreover header is a core component so I decided to put routes in this module